### DISCUSSION TITLE
CSS - Module parse failed

### DISCUSSION THREAD

**Discussion by mulkrajsingh** [vercel/next.js#66616] [Help]
### Summary

Created a nextjs 14 POC using `npx create-next-app@latest`. When I run `npm run dev` I get below error
```
./src/app/globals.css
Module parse failed: Unexpected character '@' (1:0)
> @tailwind base;
| @tailwind components;
| @tailwind utilities;
```

### Additional information

```js
Just clone the app and run `npm run dev`

os: windows 11
node: v18.20.3
npm: 10.7.0 
```

**Comment by mulkrajsingh** [comment]
It was system specific issue. I had to reset my PC to fix it.

**Comment by vyshnavsdeepak** [comment]
I faced this issue earlier and thought I should mention what I was doing wrong. I had NODE_ENV=production in my .env, which I guess was preventing Tailwind's JIT from running.

**Reply by maximilliangeorge** [reply]
Thank you, I was going crazy.

To anyone reading, just type `NODE_ENV=development` into the terminal before kicking off the dev process and it should work.

**Reply by migueldnaco** [reply]
> Thank you, I was going crazy.
> 
> To anyone reading, just type `NODE_ENV=development` into the terminal before kicking off the dev process and it should work.

OK, but what if I’m trying to deploy to production?

**Comment by JawaharAmbedkar** [comment]
i'm facing same issue , tell me if u found the solution.
